### DISCUSSION TITLE
kWh calculation

### DISCUSSION THREAD

**Discussion by ElMyggo** [UtilitechAS/amsreader-firmware#441] [Q&A] [answered]
I get strange calculation on the last 24 hour graph, hour 21 is very low and hour 22 is high but my consumption is stable, this happens sometimes.
And i have wrong factor on the graph, should be 1.26, 1.31, 1.17, 1.06 and so on..
The Real time calculation show correct value.
Maybe i have missed some settings?
![kWh](https://user-images.githubusercontent.com/111691693/218307623-e407ba03-2af8-45da-aeb5-a4e128d66327.jpg)

**Comment by gskjold** [comment, marked as answer]
Enable multipliers in meter configuration and change multiplier for "kwh" in to 0.1

**Reply by ElMyggo** [reply]
Thankyou, working now 🙂
